### DISCUSSION TITLE
Switching tickets to advanced module

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16510] [General] [answered]
Does turning tickets to advanced module will have some negative impact on mailscanner related conversion?

Best

**Comment by rskrzypczak** [comment, marked as answer]
You'll have to test it yourself, but it doesn't seem like there should be any problems as long as you configure the advanced block properly

**Reply by s2b-git** [reply]
All is OK so far
